src/pages/docs/guides/chat/build-livestream.mdx

```diff
@@ -0,0 +1,240 @@
+
+How you authenticate is also key. To balance security and experience, you want short-lived tokens that can be easily revoked if a users is misbehaving or needs their permissions changed, but also avoiding the overhead of changing permissions every 2 minutes. In production apps, we strongly recommend you don't use API keys for client-side authentication (server-side is OK) - these are long-lived and much more difficult to revoke in a hurry.
+
+With Ably Chat, authentication is best achieved using JSON Web Tokens (JWTs). These are tied to a particular clientID and come with a set of [capabilities](https://ably.com/docs/chat/setup#authentication) that control what a client can and cannot do. Ably's SDKs handle the timing and process of requesting a new token for your, refreshing it when it expires. All you need to do is provide a server-side endpoint that can generate the JWT for the client. This allows clients to use your existing authentication systems or user sessions to generate their Ably token.
+
+
+## Moderation: Protecting your community
+
+Effective moderation is essential for livestream chat to maintain a safe and engaging environment, prevent spam and inappropriate content, and ensure compliance with platform guidelines. While human moderators are valuable, they cannot scale to handle the high message volumes of popular livestreams - a single stream can generate thousands of messages per minute. Automated moderation solutions are therefore crucial for real-time content filtering at scale.
+
+Ably supports both inline (before-publish) and after-publish moderation workflows, making it easy to integrate with AI-powered moderation tools.
+
+- **After-publish moderation:** Messages appear instantly, then are removed if flagged. This minimises latency and is suitable for high-velocity chats.
+- **Before-publish moderation:** Messages are held until approved. Use this for sensitive streams where content control is paramount.
+
+### Key moderation considerations
+
+When implementing chat moderation, consider these essential factors:
+
+1. **Community standards**
+   - What level of moderation is appropriate for your audience?
+   - How will you handle different types of content? For example:
+     - **Hate speech and harassment:** Detecting discriminatory language, threats, or targeted abuse
+     - **Inappropriate content:** Flagging adult content, violence, or graphic material
+     - **Toxicity:** Measuring overall message sentiment and hostility
+   - AI moderation systems typically classify text according to categories similar to those above, usually with a severity and confidence level. If you have a provider in mind, what thresholds would you set for your chat?
+   - Do you need different rules for different user groups?
+
+2. **Technical requirements**
+   - How latency-sensitive is your chat?
+     - Inline AI moderation can add up to 100ms to message delivery
```

Comment:
Worth mentioning that you'd need to potentially factor in batching latency here as well?

Reply:
Added